source/_posts/2020-01-29-shipit-xvii.md

```diff
@@ -0,0 +1,87 @@
+---
+layout: post
+title: "ShipIt Day XVII"
+author: Caitie Oder
+date: 2020-01-29
+tags: [engineering, shipit]
+---
+
+ShipIt Day, Cerner Engineering’s 24-hour hackathon, provides associates an environment to be creative, work on new projects, and network with other associates. This event is one of the many things that sit at the core of our [Engineering Culture](https://engineering.cerner.com/blog/devculture-team) at Cerner, and is rapidly growing- now being hosted globally in Romania, Sweden, India, and the UK. In October, our team had the chance to participate in the seventeenth ShipIt Day at Kansas City’s [Innovations Campus](https://engineering.cerner.com/blog/one-year-calling-innovations-home/) and managed to take home the Golden Keyboard trophy as the first place team!
+
+<div align="center">
+  <table>
+    <tr>
+      <td align="center">
+        {% img center /assets/2020-01-29-shipit-xvii/z3ro_d4y-min.png 600px %}
+        <sub>Left to Right - Sharynne Azhar, Vu Tran, Harish Pendyala, and Jordan Bush</sub>
+      </td>
+    </tr>
+  </table>
+</div>
+
+## Overview
+During this ShipIt challenge, our team's goal is to create a dashboard that tracks newly discovered [CVEs](https://cve.mitre.org/) and displays any affected Cerner artifacts. The Z3R0 D4Y dashboard is a centralized place where teams can quickly go to identify and remediate any new security vulnerabilities and threats.
+
+## The Problem
+
+New security vulnerabilities come up every day and keeping up with the latest updates can be tough. To take a more proactive response, we needed a way to get real-time updates of the latest vulnerabilities and identify which Cerner artifacts are affected.
+
+But where can we get this data?
+
+## Our Solution: Z3R0 D4Y
+
+We created the Z3R0 D4Y dashboard (shown below) to help solve this problem.
+
+<div align="center">
+  <table>
+    <tr>
+      <td align="center">
+        {% img /assets/2020-01-29-shipit-xvii/dashboard.png %}<br>
```

Comment:
I can't believe a linter didn't pick up that this should be <br />

Reply:
good eye. i don't think this repo has a linter setup on it. I'll update to be `<br />`. It seems like both might be valid
![image](https://user-images.githubusercontent.com/8029578/73195368-5d041280-40f3-11ea-9a2c-ce8cba0b7a32.png)
but i can update to be sure